guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -0,0 +1,203 @@
+[id="Upgrading-Project-from-EL7-to-EL8{context}"]
+ifdef::satellite[]
+= Upgrading {Project} from {RHEL} 7 to {RHEL} 8
+endif::[]
+ifdef::katello,foreman-el[]
+= Upgrading {Project} from Enterprise Linux 7 to Enterprise Linux 8
+endif::[]
+
+Use this procedure to upgrade your {Project} installation from {RHEL} 7 to {RHEL} 8.
+LEAPP is the supported tool for {RHEL} 7 to {RHEL} 8 upgrades.
+
+.Prerequisites
+ifdef::katello,foreman-el[]
+* {Project} {ProjectVersionPrevious} running on CentOS Linux 7.
+endif::[]
+
+ifdef::satellite[]
+- {Project} installations running on {RHEL} 7 cannot upgrade to {RHEL} 8. You must have a {RHEL} compatible LEAPP build to upgrade.
+endif::[]
```

Comment:
This is only relevant for Foreman/Katello.
Satellite only runs on RHEL and RHEL users have the right LEAPP builds.